packages/bot-web-ui/src/components/journal/journal.jsx

```diff
@@ -247,6 +248,7 @@ const Journal = ({
                     <>
                         {contract_stage >= contract_stages.STARTING &&
                         !!props.checked_filters.length &&
+                        !unfiltered_messages.length &&
```

Comment:
Although I think here it makes no difference, but it's better to use `!!` to have the exact boolean value (Just a suggestion).